src/operations/add_user.ts

```diff
@@ -57,7 +55,9 @@ export class AddUserOperation extends CommandOperation<Document> {
     const options = this.options;
 
     // Error out if digestPassword set
-    if (options.digestPassword != null) {
+    // v5 removed the digestPassword option from AddUserOptions but we still want to throw
+    // an error when digestPassword is provided.
+    if ((options as any).digestPassword != null) {
```

Comment:
```suggestion
    if ('digestPassword' in options && options.digestPassword != null) {
```

Time to bump to TS 4.9? (just a suggestion no need to do that here, would just be nice to avoid this casting)

Reply:
🤔 Nice idea.  I'll try it and if it doesn't cause problems, I'll make the change